x/checkpoint/client/cli/query.go

```diff
@@ -1,30 +1,173 @@
 package cli
 
 import (
+	"context"
 	"fmt"
-	// "strings"
 
+	"github.com/gogo/protobuf/codec"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
```

Comment:
@temaniarpit27 was mentioning we are not going to be using viper, check clerk module pr to get the flags without the use of viper.